.github/workflows/ruby.yml

```
@@ -6,8 +6,8 @@ env:
   POSTGRES_USER: postgres
   POSTGRES_PASSWORD: postgres
   POSTGRES_HOST: localhost
-  POSGRES_DB: textie
-  DATABASE_URL: "postgresql://postgres:postgres@localhost:5432/textie"
+  POSGRES_DB: textie_test
```

```suggestion
  POSTGRES_DB: textie_test
```

Should we rename it?